source/concurrent/Thread.ooc

```diff
@@ -12,19 +12,39 @@ import native/[ThreadUnix, ThreadWin32]
 
 Thread: abstract class {
 	_code: Func
+	_canceled := false
+	_cancelable: Bool
+	_cancelMutex: Mutex = null
+	_ownsCancelMutex: Bool
```

Comment:
OK, great. I see this does not even have the PR flag, I won't bother you again:)